chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eo pipefail
+# Args check
+if [[ "$#" -ne 2 ]]; then
+    echo "Error: Two arguments are required"
+    exit 1
+fi
+# Function for delimiter
+print_separator() {
```

Comment:
Please reorganize code like this:

```
func1 ()
func2 ()

if [[ "$#" -ne 2 ]]; then
  echo "Error: Two arguments are required"
  exit 1
fi
...
...
```

So functions would be always above and the code, which is not wrapped in functions would be at the end. So we won't have situation that you have some codes between funcs